core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -579,17 +591,15 @@ fi
 #####################################################
 
 # See JIRA TRAFODION-1512 that will eliminate this check
-if [ -d $MY_LOCAL_SW_DIST ]; then
   
-  HADOOP_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
-  HADOOP_TAR=hadoop-2.5.0-cdh5.3.0.tar.gz
-else
-  HADOOP_MIRROR_URL=http://download.nextag.com/apache/hadoop/common/hadoop-2.5.2
-  HADOOP_TAR=hadoop-2.5.2.tar.gz
-fi
+HADOOP_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
+HADOOP_TAR=hadoop-2.6.0-cdh5.4.4.tar.gz
 
-if [[ "$SQ_HBASE_DISTRO" = "HDP" ]]; then
-    HADOOP_TAR=hadoop-2.6.0.2.2.0.0-2041.tar.gz
+if [[ "$HBASE_DISTRO" = "HDP" ]]; then
```

Comment:
Is this a new requirement, that we set HBASE_DISTRO to some value before running install_local_hadoop? Maybe this is optional, only if we want to get something other than CDH? If so, that sounds good. We can make Apache the default once we have tested it for a while.